libs/community/langchain_community/agent_toolkits/nla/tool.py

```diff
@@ -13,7 +13,7 @@
 from langchain_community.utilities.requests import Requests
 
 
-class NLATool(Tool):
+class NLATool(Tool):  # type: ignore[override]
```

Comment:
ooc what is driving the need for the new `# type: ignores`? is it mypy getting updated?

Reply:
Should we instead `@override` overridden methods? I believe that might be what it's trying to enforce

Reply:
(happy to merge this now and do a sprint on fixing ignores like this)

Reply:
@ccurme im not actually sure, afaict it was same mypy version. but issue was only happening in py3.13 so maybe im missing something